internal/compiler/langtype.rs

```diff
@@ -546,7 +546,7 @@ impl ElementType {
                         if !tr.expose_internal_types
                             && matches!(&t, Self::Builtin(e) if e.is_internal)
                         {
-                            format!("Unknown element '{name}'. (The type exist as an internal type, but cannot be accessed in this scope)")
+                            format!("Unknown element '{name}'. (The type exists as an internal type, but cannot be accessed in this scope)")
```

Comment:
This change causes the test to fail because we test for this errors in the  internal/compiler/tests/syntax tests.
Please also update the test to match the right error.